app/controllers/pets_controller.rb

```diff
@@ -40,6 +40,18 @@ def destroy
     redirect_to "/pets"
   end
 
+  def pending
+    pet = Pet.find(params[:id])
+    pet.update!(adoption_status: false)
+    redirect_to "/pets/#{pet.id}"
+  end
+
+  def adoptable
+    pet = Pet.find(params[:id])
+    pet.update!(adoption_status: true)
+    redirect_to "/pets/#{pet.id}"
+  end
```

Comment:
I wonder if this is too repetitive and I could just make them one method or put them into #update.